spec/restaurant_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'restaurant' 
+
+RSpec.describe Restaurant do
+  
+  describe '#show_menu' do
+    it 'shows the menu of the restaurant' do
+      expect(subject.show_menu).to eq(create_example_menu)
+    end
+  end
+
+  describe '#create_order' do
+    it 'creates a new Order-Object and returns it' do
+      expect(subject.create_order).to be_a(Order)
+    end
+  end
+
+  describe '#add_dish_to_order' do
+    it 'adds dishes to an order' do
+      subject.create_order()
+      subject.add_dish_to_order('Chips')
+      subject.add_dish_to_order('Chips')
+      subject.add_dish_to_order('I am not in the the menu')
```

Comment:
Yeah, not sure why I actually didn't do it, I remember that it would be an edge case if something is ordered, that is not in the menu and I decided, that the program would just simply ignore it, and doesn't break. But at least it should tell the user... this was actually the first time twilio SMS, probably this would be a great opportunity to try twilio sending a MMS with a RTFM picture :)